### ISSUE TITLE
Conflict with Nature Arise causing crash/freeze

### ISSUE THREAD

**Issue by Gamemassa** [jellysquid3/Lithium#559]
### Version Info
Minecraft 1.21
Fabric Loader 0.16.0 (and 0.16.3)
Fabric API 0.100.7 (and 0.102.0)
Lithium 0.12.7 (and 0.13.0) for MC 1.21
Nature Arise 0.9.1 (and 0.9.2) for MC 1.21

### Reproduction Steps
Trying to open the creative inventory with both Lithium 0.12.7 and Nature Arise 0.9.1 together causes the game to crash to desktop. Upgrading Lithium to 0.13.0 (which was necessary after updating Fabric to 0.16.3 and Fabric API to 0.102.0) and Nature Arise to 0.9.2, the game no longer crashes, but still freezes and has to be forcibly closed.

### Crash Report Files

- With Lithium 0.12.7 and Nature Arise 0.9.1

   - Crash Report:  
      https://mclo.gs/sDW7oGl
   - Latest.log: 
     https://mclo.gs/bectrTA

- With Lithium 0.13.0 and Nature Arise 0.9.2
    - Crash Report
N/A

    - Latest.log
 https://mclo.gs/xPlBX7l

**Comment by 2No2Name** [comment]
The 0.13.0 issue does not look like it is related to lithium. Can/Did you test that version without lithium?

**Comment by 2No2Name** [comment]
The 0.12.7 issue also looks like it is unrelated to lithium

**Comment by Gamemassa** [comment]
> The 0.13.0 issue does not look like it is related to lithium. Can/Did you test that version without lithium?

I did do separate tests with Lithium alone and with Nature Arise alone, and the game didn't crash/freeze in any of those tests.

Out of curiosity, what did you see in the crash report that tells you they may not be related?

**Comment by 2No2Name** [comment]
In the crash report there is nothing that hints at lithium being a cause. The exceptions are thrown in code that lithium doesn't touch at all

**Comment by Gamemassa** [comment]
> In the crash report there is nothing that hints at lithium being a cause. The exceptions are thrown in code that lithium doesn't touch at all

Well it's definitely caused by Lithium and Nature Arise both being installed, so that's incredibly weird.

**Comment by IThundxr** [comment]
Nature's arise isn't class loading their blocks class at all, which from the looks means that by the time the creative tab event runs it's blocks can possibly not be registered/the registries could be frozen

**Comment by Gamemassa** [comment]
> Nature's arise isn't class loading their blocks class at all, which from the looks means that by the time the creative tab event runs it's blocks can possibly not be registered/the registries could be frozen

When you say "creative tab event" are you referring to some feature of Lithium? Or just, is there something that Lithium does in relation to  the Creative Tab?

**Comment by IThundxr** [comment]
No it's a fabric api provided event to add entries to creative mode tabs

**Comment by 2No2Name** [comment]
My best guess is that you can try disabling the nbt mixin:

Add the line `mixin.alloc.nbt=false` to the lithium.properties file in the config directory in the .minecraft folder

**Comment by Gamemassa** [comment]
Oh, thanks! I will try that, and I'll let you know how it goes.

**Comment by Gamemassa** [comment]
> My best guess is that you can try disabling the nbt mixin:
> 
> Add the line `mixin.alloc.nbt=false` to the lithium.properties file in the config directory in the .minecraft folder

I tried adding that to the lithium.properties file, and now when I access the Creative Inventory, I get this crash:
- Crash Report: https://mclo.gs/1tornV3
- Latest.log: https://mclo.gs/Cd1YTgU

**Comment by 2No2Name** [comment]
That didn't make any difference then

**Comment by Gamemassa** [comment]
> My best guess is that you can try disabling the nbt mixin:
> 
> Add the line `mixin.alloc.nbt=false` to the lithium.properties file in the config directory in the .minecraft folder

Ok, gonna maybe sound dumb for a second, but would it change anything if there are spaces before and after the = ?

**Comment by 2No2Name** [comment]
I never tested it with spaces, so I don't know

**Comment by Gamemassa** [comment]
> I never tested it with spaces, so I don't know

Yeah, spaces didn't change anything. Also, if I'm reading that last crash report correctly, Nature Arise is trying to add an empty stack of something but I don't know what.
